packages/suite/src/components/suite/HomescreenGallery/index.tsx

```diff
@@ -62,7 +62,7 @@ const HomescreenGallery = ({ device, onConfirm }: Props) => {
 
     return (
         <Wrapper>
-            {!isModelT && (
+            {isModel1 && (
```

Comment:
nit: perhaps instead of a ternary it makes sense to have a render like:

```
 {isModel1 && <>stuff</>}
 {(isModelT || isModelR) && <>stuff</>}
```

Or even introducing model grouping with one group having model 1 and the other T&R.

Reply:
yes, but i do not want to do it in this hotfix branch